lib/validators/movie.js

```diff
@@ -3,6 +3,6 @@
 const Joi = require('joi');
 
 module.exports = Joi.object().keys({
-  title: Joi.string().min(1).max(255).required(),
+  name: Joi.string().min(1).max(255).required(),
```

Comment:
in reality this probably wouldnt change just like this. if it was going to actually change, youd do it in an api version. this is fine for now.

Reply:
Specifically in the Joi validator or everywhere?